src/index.js

```diff
@@ -20,13 +20,13 @@ import "react-app-polyfill/stable";
 import React from "react";
 import ReactDOM from "react-dom";
 import App from "./App";
-import data from "./data.json";
+import data from "./data-zero.json";
```

Comment:
Let's get rid of `data-zero.json` from this repository and revert back to `data.json` before we merge. With this change, it is actually a bit trickier to test this locally because it makes it seem like you've already taken care of Colin's steps to reproduce the bug but it's not 100% clear if that is the case. So reverting back to CO and back to the original data file will make it so I know exactly what to do to test this.